data/news/articles/c0g992jmmkko.json

```diff
@@ -895,11 +895,12 @@
       "metadata": {
         "id": "urn:bbc:ares::article:c0g992jmmkko",
         "atiAnalytics": {
-          "categoryName": "Wedding%20of%20Prince%20Harry%20and%20Meghan%20Markle~Duchess%20of%20Sussex~Hackney~Windsor",
+          "categoryName": "Wedding+of+Prince+Harry+and+Meghan+Markle~Duchess+of+Sussex~Hackney~Windsor",
```

Comment:
Does getting rid of the space encoding here affect anything?

Reply:
This was confirmed to have no effect when we migrated the Optimo
article page. I'l check with Andrew Anderson to confirm the same applies to CPS assets.